atcoder-problems-frontend/src/pages/Internal/VirtualContest/ShowContest/ContestTable.tsx

```diff
@@ -288,6 +289,17 @@ export const ContestTable = (props: Props) => {
             />
           );
         })}
+        {showProblems ? (
+          <FirstAcceptedRow
+            start={start}
+            userIds={users}
+            problemIds={sortedItems.map((item) => {
+              return item.id;
+            })}
+            resultsByUser={resultsByUser}
+            showRating={showRating}
+          />
+        ) : null}
```

Comment:
```suggestion
        {showProblems && (
          <FirstAcceptedRow
            start={start}
            userIds={users}
            problemIds={sortedItems.map((item) => {
              return item.id;
            })}
            resultsByUser={resultsByUser}
            showRating={showRating}
          />
        )}
```
実はbooleanがfalseの時は無なので、このように簡略化できます。